tailwind.config.js

```diff
@@ -19,6 +19,33 @@ module.exports = {
         serif: ["var(--font-sora)", ...fontFamily.serif],
         scribble: ["var(--font-scribble)", ...fontFamily.mono],
       },
+      keyframes: {
+        wave: {
+          "0%": { transform: "rotate(0.0deg)" },
+          "10%": { transform: "rotate(14deg)" },
+          "20%": { transform: "rotate(-8deg)" },
+          "30%": { transform: "rotate(14deg)" },
+          "40%": { transform: "rotate(-4deg)" },
+          "50%": { transform: "rotate(10.0deg)" },
+          "60%": { transform: "rotate(0.0deg)" },
+          "100%": { transform: "rotate(0.0deg)" },
+        },
+        growandshrink: {
```

Comment:
Can you try and rename it using snake case? To be more consistent with other animation classes.

```suggestion
        grow-and-shrink: {
```

Reply:
It works with underscore grow_and_shrink